Userland/Utilities/pidof.cpp

```diff
@@ -38,8 +40,13 @@ static int pid_of(const String& process_name, bool single_shot, bool omit_pid, p
     return 0;
 }
 
-int main(int argc, char** argv)
+ErrorOr<int> serenity_main(Main::Arguments args)
 {
+    TRY(Core::System::pledge("stdio rpath"));
+    TRY(Core::System::unveil("/proc/all", "r"));
+    TRY(Core::System::unveil("/etc/passwd", "r"));
+    TRY(Core::System::unveil(nullptr, nullptr));
```

Comment:
The `pledge()` and `unveil()` calls are new additions. I think it is worth to mention it in the commit or split it out in a separate commit